WalletWasabi/Helpers/Constants.cs

```diff
@@ -76,21 +73,6 @@ public static class Constants
 			MinProtocolCapabilities = new ProtocolCapabilities { SupportGetBlock = true, SupportWitness = true }
 		};
 
-		public static readonly NodeRequirement LocalBackendNodeRequirements = new NodeRequirement
-		{
-			RequiredServices = NodeServices.NODE_WITNESS,
-			MinVersion = ProtocolVersionWitnessVersion,
-			MinProtocolCapabilities = new ProtocolCapabilities
-			{
-				SupportGetBlock = true,
-				SupportWitness = true,
-				SupportMempoolQuery = true,
-				SupportSendHeaders = true,
-				SupportPingPong = true,
-				PeerTooOld = true
-			}
-		};
```

Comment:
Why is it unused?

Reply:
I have no idea why it is not used.

Reply:
Would you mind tracking it down? I'd like to make sure it's intentional.

Reply:
It was added in https://github.com/zkSNACKs/WalletWasabi/pull/1216/files and was removed later in https://github.com/zkSNACKs/WalletWasabi/pull/2546/files

Reply:
Thanks, I don't know what I did there, but I'm sure I didn't do it accidentally.